distribution/src/root/config/examples/hazelcast-security-hardened.yaml

```diff
@@ -0,0 +1,262 @@
+# This is a sample Hazelcast configuration focused on hardened security.
```

Comment:
I think the file should go to `hazelcast/src/main/resources` so it is also in the main jar and next to other example files and be added to the distribution in the assembly descriptors (both regular and slim). At least that's how @mmedenjak wanted it.

Reply:
Yes, right, all in one place and in the examples directory in the zip/tar :)